examples/bank/bank.go

```diff
@@ -93,18 +96,84 @@ func (bank *Bank) moveMoney(from, to []byte, amount int64) bool {
 			return errRead
 		}
 		// Update both accounts.
-		txn.Prepare(client.Put(proto.Key(from), []byte(fmt.Sprintf("%d", fromValue-amount))))
-		txn.Prepare(client.Put(proto.Key(to), []byte(fmt.Sprintf("%d", toValue+amount))))
+		txn.Prepare(client.Put(proto.Key(from), []byte(fmt.Sprintf("%d", fromValue-amount))),
```

Comment:
I think I asked before but I don't remember if I got an answer: You're deliberately not using Increment and our Integer types, right?